### DISCUSSION TITLE
NDI Polaris Vega XT api source files

### DISCUSSION THREAD

**Discussion by D49ll** [PlusToolkit/PlusLib#1121] [Q&A] [answered]
Hi,

Im in the process of trying to compare the NDI Polaris Vega XT to my master's thesis OTS. I am just wondering what ".cxx" belongs to the NDI Polaris system, is it  NDICAPITracking or NDICertusTracking. I am asking because the new system has to have the same functionalites as the Polaris system.

**Comment by adamrankin** [comment, marked as answer]
https://github.com/PlusToolkit/PlusLib/blob/master/src/PlusDataCollection/NDICAPITracking/vtkPlusNDITracker.cxx
